### PR TITLE
Feature/kafka ingestion

### PR DESCRIPTION
**Pull request by dmcollom**

### CODE REVIEW

File: ingest.py [hunk 1]
```diff
@@ -1,53 +1,45 @@
...
 from astropy.time import Time
 
 from ztf import Alert, db, app
 
-REDIS_HOST = os.getenv('REDIS_HOST', '127.0.0.1')
-r = redis.StrictRedis(host=REDIS_HOST, charset='utf-8', decode_responses=True)
-redis_broker = RedisBroker(url=f'redis://{REDIS_HOST}:6379/0')
-dramatiq.set_broker(redis_broker)
-
-AWS_ACCESS_KEY = os.getenv('AWS_ACCESS_KEY_ID')
```
Comment (Fingel) [inline ingest.py]: I know boto checks these env variables behind the scenes on it's own, but as someone that doesn't know that I'd be pretty confused as to how to set the credentials. Can we leave these in to be explicit?

File: ingest.py [hunk 2]
```diff
@@ -81,9 +73,14 @@ def ingest_avro(packet):
             gal_b=galactic.b.value,
             **packet['candidate']
             )
-        db.session.add(alert)
-        db.session.commit()
-        logger.info(alert.objectId)
+        try:
+            db.session.add(alert)
+            db.session.commit()
+            logger.info('Inserted object %s', alert.objectId)
```
Comment (Fingel) [inline ingest.py]: Now that we know candid is unique, let's use that in lieu of objectId where we can.

File: ingest.py (hunk 1 above)
Comment (dmcollom) [reply ingest.py]: Yeah, makes sense. Figured I'd play defense on those, but you're right, it's clearer.

File: ingest.py (hunk 2 above)
Comment (dmcollom) [reply ingest.py]: Done.

**Pull request merged**